pyproject.toml

```diff
@@ -4,6 +4,12 @@ src_paths = ["exporters" ,"scripts"]
 known_first_party = ["pelorus"]
 
 [tool.pytest.ini_options]
+pythonpath = [
+    "exporters/comitttime",
+    "exporters/deploytime",
+    "exporters/failure",
+    "exporters/pelorus"
+]
```

Comment:
I'd rather we make `failure` a proper module so we can just change the imports to `from failure.$submodule ...`. Seems more proper that way, right? If you agree, I can push the small change to the PR.

Reply:
@KevinMGranger  Ok, please do.